### PR TITLE
Prepare for 7.2.6 Patch Release

### PR DESCRIPTION
**Pull request by PingXie**

### CODE REVIEW

File: 00-RELEASENOTES [hunk 1]
```diff
@@ -12,6 +12,41 @@ SECURITY: There are security fixes in the release.
 --------------------------------------------------------------------------------
 
 ================================================================================
+Valkey 7.2.6 GA  -  Released Tue 30 July 2024
+================================================================================
+
+Upgrade urgency MODERATE: Program an upgrade of the server, but it's not urgent.
+
+Bug fixes - Core
+=========
```
Comment (madolson) [inline 00-RELEASENOTES]:
```suggestion
Bug fixes - Core
================
```

File: 00-RELEASENOTES [hunk 2]
```diff
@@ -12,6 +12,41 @@ SECURITY: There are security fixes in the release.
...
+* Fix the command duration reset issue when clients are blocked and commands are reprocessed (#526)
+* Fix the data type conversion error in zrangeResultBeginStore (Redis#13148)
+* Fix a crash caused by quicklist node merges (Redis#13040)
+* Fix crashes in module blocking client timeout cases (Redis#13011)
+* Fix conversion of numbers in Lua args to Redis args (Redis#13115, Fixes Redis#13113)
+* Fix crash in LSET command when replacing small list items with larger ones, creating listpacks larger than 4GB (Redis#12955, Fixes Redis#12864)
+* Fix blocking command timeout reset issue during reprocessing (Redis#13004) 
+
+Bug fixes - Cluster
+=========
```
Comment (madolson) [inline 00-RELEASENOTES]:
```suggestion
Bug fixes - Cluster
===================
```

File: 00-RELEASENOTES [hunk 3]
```diff
@@ -12,6 +12,41 @@ SECURITY: There are security fixes in the release.
...
+* Ensure only primary nodes with slots can mark another node as failed (#634)
+* Improve MEET command reliability under link failures to maintain cluster membership symmetry (#461)
+* Allow single primary node to mark potentially failed replica as FAIL in single-shard cluster (Redis#12824)
+
+Bug fixes - Sentinel
+=========
+* Accept redis-sentinel to start Valkey in sentinel mode (#731, Fixes #719)
+
+Bug fixes - CLI
+=========
```
Comment (madolson) [inline 00-RELEASENOTES]:
```suggestion
Bug fixes - Sentinel
====================
* Accept redis-sentinel to start Valkey in sentinel mode (#731, Fixes #719)

Bug fixes - CLI
===============
```

File: 00-RELEASENOTES [hunk 4]
```diff
@@ -12,6 +12,41 @@ SECURITY: There are security fixes in the release.
 --------------------------------------------------------------------------------
 
 ================================================================================
+Valkey 7.2.6 GA  -  Released Tue 30 July 2024
+================================================================================
+
+Upgrade urgency MODERATE: Program an upgrade of the server, but it's not urgent.
```
Comment (madolson) [inline 00-RELEASENOTES]:
```suggestion
Upgrade urgency Medium: This release fixes an incompatibility issue with modules compiled for Redis. For other users, it primarily is fixing uncommon bugs.
```
Normally we add more context about the upgrade here. I think the biggest thing is the module compatibility, everything else is just a minor improvement.

File: 00-RELEASENOTES [hunk 5]
```diff
@@ -12,6 +12,41 @@ SECURITY: There are security fixes in the release.
...
+
+Bug fixes - Core
+=========
+* Fix typo in REGISTER_API macro to prevent segfaults when loading Redis modules (#608)
+* Fix the command duration reset issue when clients are blocked and commands are reprocessed (#526)
+* Fix the data type conversion error in zrangeResultBeginStore (Redis#13148)
+* Fix a crash caused by quicklist node merges (Redis#13040)
+* Fix crashes in module blocking client timeout cases (Redis#13011)
+* Fix conversion of numbers in Lua args to Redis args (Redis#13115, Fixes Redis#13113)
+* Fix crash in LSET command when replacing small list items with larger ones, creating listpacks larger than 4GB (Redis#12955, Fixes Redis#12864)
```
Comment (madolson) [inline 00-RELEASENOTES]: The rest of the file reflows the lines after 80 characters, I don't really care but maybe we should be consistent.

File: 00-RELEASENOTES [hunk 6]
```diff
@@ -12,6 +12,41 @@ SECURITY: There are security fixes in the release.
...
+=========
+* Accept redis-sentinel to start Valkey in sentinel mode (#731, Fixes #719)
+
+Bug fixes - CLI
+=========
+* Ensure the --count option in redis-cli works correctly even without --pattern (Redis#13092)
+* Fix redis-check-aof misidentifying data in manifest format as MP-AOF (Redis#12951)  
+* Update redis-check-rdb types to replace stream-v2 with stream-v3 (Redis#12969)
+
+=================================================================================
```
Comment (madolson) [inline 00-RELEASENOTES]:
```suggestion
================================================================================
```

**Review by madolson — COMMENTED** [review]
Mostly nitpicky style stuff, the content seems fine.

File: 00-RELEASENOTES (hunk 5 above)
Comment (PingXie) [reply 00-RELEASENOTES]: What about reflowing around 120 characters? PTAL

Comment (PingXie) [reply 00-RELEASENOTES]: never mind. 120 looks a bit odd. will do 80.

Comment (PingXie) [reply 00-RELEASENOTES]: done

**Review by madolson — APPROVED** [review]

**Pull request merged**